### PR TITLE
Release v1.6.0

### PR DESCRIPTION
**Pull request by araffin**
## Description

 - Update doc (add copy button, add example for envpool/isaac gym, add link for using latest gym version).
 - Fix bug in kl div check, see SB3 contrib: https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/issues/83

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [ ] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [ ] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format` (**required**)
- [ ] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [ ] I have ensured `make pytest` and `make type` both pass. (**required**)
- [ ] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: docs/guide/examples.rst [hunk 1]
```diff
@@ -729,6 +729,16 @@ to keep track of the agent progress.
   model.learn(10_000)
 
 
+SB3 with EnvPool or Isaac Gym
+-----------------------------
+
+Just like Procgen (see above), `EnvPool <https://github.com/sail-sg/envpool>`_ and `Isaac Gym <https://github.com/NVIDIA-Omniverse/IsaacGymEnvs>`_ accelerate the environment by
+already providing a vectorized implementation.
+
+To use SB3 with those tools, you must wrap the env with a ``VecEnvWrapper`` that will pre-process the data for SB3,
```
Comment (Miffyli) [inline docs/guide/examples.rst:738]: Hmm shouldn't the instruction say that to use SB3 with those tools, you need to wrap it with the tool's version of ``VecEnvWrapper`` (i.e. this `VecEnvWrapper` is not part of SB3)? It currently sounds like it is part of SB3 and you need to use it on top of whatever the tool will provide.

**Review by Miffyli — CHANGES_REQUESTED** [review]
LGTM apart from one small adjustement to the docs :)

File: docs/guide/examples.rst (hunk 1 above)
Comment (araffin) [reply docs/guide/examples.rst:738]: true, I should be more specific

**Review by Miffyli — APPROVED** [review]
LGTM :)

**Pull request merged**